### ISSUE TITLE
No validation loss shown in tensorboard

### ISSUE THREAD

**Issue by oathaha** [antonio-mastropaolo/T5-learning-ICSE_2021#2]
While I was fine-tuning T5 model, I found that there was no validation loss being shown in tensorboard. So I have no idea when to stop fine-tuning. Can you tell me how to track validation loss along with train loss, and when to stop fine-tuning phase?

Thank you.

**Comment by antonio-mastropaolo** [comment]
Hello,
I didn't use tensorboard to keep track of the losses. Instead, I have computed specifics metric each #K checkpoints (\ie early stopping). You could do the same.
If you want to use tensorboard, try to have a look at this: https://github.com/google-research/text-to-text-transfer-transformer/blob/main/notebooks/t5-trivia.ipynb
